_posts/2018-09-12-september-office-hours.md

```diff
@@ -0,0 +1,72 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-11
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+author_staff_member: burton
+dark_background: true
+---
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as specialized discussions with project members, and core contributors. During these discussions, we talk about what's new in the project, where we're headed, and any other updates individual members may have.
+
+All of the office hours meetings are recorded and available for later viewing on YouTube.
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/gkKpercV6FA?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+> We also maintain an [official channel](https://www.youtube.com/channel/UCdKi97g5FmzvrmtIp9FyOVA) that contains demos from some of the meetings
```

Comment:
Instead of: We maintain an official channel.

Subscribe to the Official OpenFaaS YouTube Channel where you can view the highlights from our meetings including demos of the newest features we're working on.